### PR TITLE
FIx markdown link and 'guide instance' translation improvements

### PR DESCRIPTION
**Pull request by MatiasVerdier**
I've found this issue in the 'instance' page

Screen Shot 2019-05-04 at 7 09 55 PM

I added some missing translations for code comments and I'm proposing some possible improvements to the current translation.

### CODE REVIEW

File: src/v2/guide/instance.md [hunk 1]
```diff
@@ -10,15 +10,14 @@ Cada aplicación de Vue se comienza creando una nueva **Instancia de Vue** con l
...
 ```js
 var vm = new Vue({
-  // options
+  // opciones
 })
 ```
 
-Aunque no extrictamente asociado con el [patrón MVVM]
-(https://en.wikipedia.org/wiki/Model_View_ViewModel), el diseño de Vue fue en parte inspirado por él.
+Aunque no extrictamente asociado con el [patrón MVVM](https://en.wikipedia.org/wiki/Model_View_ViewModel), el diseño de Vue fue en parte inspirado por él.
```
Comment (FLarra) [inline src/v2/guide/instance.md]: There is a typo here, it should be:

```suggestion
Aunque no estrictamente asociado con el [patrón MVVM](https://en.wikipedia.org/wiki/Model_View_ViewModel), el diseño de Vue fue en parte inspirado por él.
```

File: src/v2/guide/instance.md [hunk 2]
```diff
@@ -116,17 +115,17 @@ var vm = new Vue({
 vm.$data === data // => true
 vm.$el === document.getElementById('example') // => true
 
-// $watch is an instance method
+// $watch es un metodo de instancia
```
Comment (FLarra) [inline src/v2/guide/instance.md]: There is another typo here, it should be:

```suggestion
// $watch es un método de instancia
```

File: src/v2/guide/instance.md [hunk 3]
```diff
@@ -116,17 +115,17 @@ var vm = new Vue({
...
 vm.$watch('a', function (newValue, oldValue) {
-  // This callback will be called when `vm.a` changes
+  // Esta funcion sera ejecutada cuando `vm.a` cambie
```
Comment (FLarra) [inline src/v2/guide/instance.md]: Please be careful with the misspellings, there it should be:

```suggestion
  // Esta función será ejecutada cuando `vm.a` cambie
```

**Review by FLarra — COMMENTED** [review]
If you are editing in VS Code, could be helpful to use some spell checker. 

In my case, I installed the plugin https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker-spanish and it's helpful to reduce the misspellings. It's not perfect, but in many cases identifies issues

File: src/v2/guide/instance.md (hunk 1 above)
Comment (MatiasVerdier) [reply src/v2/guide/instance.md]: Thanks for the suggestion, this was already in the code and I didn't saw it when I fixed the markdown issue. But the missing accent marks are completely my fault haha ;)

**Review by sxcamacho — APPROVED** [review]

**Pull request merged**